### PR TITLE
Open threads from PR #149

### LINKED ISSUE #155: Assets for sale list: Show only first 10 assets for sale
It's a temporary solution.
See more: https://github.com/tasitlabs/tasit/issues/154

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#155 
https://github.com/tasitlabs/tasit/pull/149#discussion_r266663781
https://github.com/tasitlabs/tasit/pull/149#discussion_r266668086
https://github.com/tasitlabs/tasit/pull/149#discussion_r266670722
https://github.com/tasitlabs/tasit/pull/149#discussion_r266672496

### Auto-close the issue?

Closes #155 

### Types of changes

New feature (non-breaking change that adds functionality)

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

File: decentraland/components/presentational/Estate.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ Estate.propTypes = {
 
 const styles = StyleSheet.create({
   landImage: {
-    width: responsiveWidth(60),
+    width: responsiveWidth(95),
     height: responsiveHeight(20),
```
Comment (pcowgill) [inline decentraland/components/presentational/Estate.js]: There weren't any height changes required?

File: decentraland/package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "android": "npx expo start --android",
     "ios": "npx expo start --ios",
     "eject": "npx expo eject",
+    "pretest": "npm run lint",
```
Comment (pcowgill) [inline decentraland/package.json]: Good call

File: decentraland/screens/BuyLandScreen.js [hunk 3]
```diff
@@ -76,7 +76,10 @@ export class BuyLandScreen extends React.Component {
       // TODO: This function should be called inside of the eventListener
       // that catches the safeExecuteOrder successful event.
       await action.waitForNonceToUpdate();
-      showInfo(`Asset bought successfully.`);
+
+      if (type == ESTATE) showInfo(`Estate bought successfully.`);
+      else if (type == PARCEL) showInfo(`Parcel bought successfully.`);
```
Comment (pcowgill) [inline decentraland/screens/BuyLandScreen.js]: Should we do an else to log an error? I like to handle all possible cases whenever possible

File: decentraland/screens/helpers.js [hunk 4]
```diff
@@ -65,73 +61,12 @@ export const addressesAreEqual = (address1, address2) => {
...
-      id: parcelId,
-      name,
-      img: imgUrl,
-    },
-  };
-};
-
 export const fundAccount = async accountAddress => {
-  const SMALL_AMOUNT = `${1e17}`; // 0.1
+  const SMALL_AMOUNT = `${5e16}`; // 0.1
```
Comment (pcowgill) [inline decentraland/screens/helpers.js]: Is the 0.1 comment still relevant?

File: decentraland/components/presentational/Estate.js (hunk 1 above)
Comment (marcelomorgado) [reply decentraland/components/presentational/Estate.js]: For me, the way that it now likes good.

File: decentraland/screens/BuyLandScreen.js (hunk 3 above)
Comment (marcelomorgado) [reply decentraland/screens/BuyLandScreen.js]: OK, done.

File: decentraland/screens/helpers.js (hunk 4 above)
Comment (marcelomorgado) [reply decentraland/screens/helpers.js]: I've just updated that.

Comment (pcowgill) [reply decentraland/screens/helpers.js]: Thanks! Resolving

File: decentraland/screens/BuyLandScreen.js (hunk 3 above)
Comment (pcowgill) [reply decentraland/screens/BuyLandScreen.js]: Thanks! Resolving

File: decentraland/components/presentational/Estate.js (hunk 1 above)
Comment (pcowgill) [reply decentraland/components/presentational/Estate.js]: Okay, sounds good. I'll take a look the next time I run it locally. Resolving this.

File: decentraland/package.json (hunk 2 above)
Comment (pcowgill) [reply decentraland/package.json]: Resolving this

**Pull request merged**